ddev/ddev
# fix: PHP CodeSniffer on settings.ddev.php for Drupal 8 - 11

## The Issue

The `settings.ddev.php` file from Drupal versions 8 to 11 had various PHP CodeSniffer violations. 

## How This PR Solves The Issue

Correcting these PHP CodeSniffer violations using default Drupal rules ensures our Drupal configurations adhere to standard code quality norms and fix the issues. 

## Manual Testing Instructions
* Copy those files to your Drupal Composer project
* Ensure PHP CodeSniffer ([drupal/coder](https://packagist.org/packages/drupal/coder)) with Drupal rules installed
* Run phpcs to validate the `settings.ddev.php` files.

## Automated Testing Overview

This PR does not introduce new functionality that will be covered by existing automated tests. If not, let me know, so we can create a test for it.
